data_structures/heap/skew_heap.py

```diff
@@ -21,14 +21,37 @@ def __init__(self, value: T) -> None:
 
     @property
     def value(self) -> T:
-        """Return the value of the node."""
+        """
+        Return the value of the node.
+
+        >>> sn = SkewNode(10)
```

Comment:
"""
Return the value of the node.

>>> sn = SkewNode(0)
>>> sn.value
0

>>> sn = SkewNode(3.14159)
>>> sn.value
3.14159

>>> sn = SkewNode("hello")
>>> sn.value
'hello'

>>> sn = SkewNode(None)
>>> sn.value
None

>>> sn = SkewNode(True)
>>> sn.value
```suggestion
        >>> sn = SkewNode(10)
```
```suggestion
        >>> sn = SkewNode(10)
```
True

>>> sn = SkewNode([])
>>> sn.value
[]

>>> sn = SkewNode({})
>>> sn.value
{}

>>> sn = SkewNode(set())
>>> sn.value
set()

>>> sn = SkewNode(0.0)
>>> sn.value
0.0

>>> sn = SkewNode(-1e-10)
>>> sn.value
-1e-10
"""